app/services/rapid_ready_for_decision/views/shared/medications.erb

```diff
@@ -9,13 +9,14 @@
   <i>All VAMC locations using VistA/CAPRI were checked</i> <br />
 </p>
 
-<% unless medications? %>
+<% unless @assessed_data[:medications].any? %>
```

Comment:
If `@assessed_data[:medications]` is nil, this will raise an error. I suggest creating a test so that we handle that situation.

Reply:
Oof, thank you. Okay!

Reply:
@yoomlam how about `present?`? It checks blank arrays and works on `nil`.

Reply:
I'd need to update this template to something like... any preference?

```erb
<% if @assessed_data[:medications].present? %>
  <% @assessed_data[:medications].each do |medication| %>
    <p>
      <b><%= medication['description'] %></b> <br />
      Prescribed on: <%= medication['authoredOn'][0, 10].to_date.strftime('%m/%d/%Y') %> <br />
      <% if medication['dosageInstructions'].present? %>
        Dosage instructions: <%= medication['dosageInstructions'].join('; ') %><br />
      <% end %>
    </p>
  <% end %>
<% else %>
  <p><br/></p>
  <h6>No active medications were found in the last year</h6>
<% end %>
```

Reply:
Gonna push up a change with a test. Let me look at how this might apply elsewhere in the code...

Reply:
Looks good!